src/suggestFix.ts

```diff
@@ -32,7 +32,7 @@ const getCommentStringByFileName = (file: string): string => {
     return specialFilesDict[fileType] || '#';
 };
 
-const generateSkipComment = (checkId: string, file: string) => `\t${getCommentStringByFileName(file)} checkov:skip=${checkId}: ADD REASON\n`;
+const generateSkipComment = (checkId: string, file: string) => `${getCommentStringByFileName(file)} checkov:skip=${checkId}: ADD REASON\n`;
```

Comment:
Can we make it 2 spaces instead?

Reply:
@tsmithv11 sure replaced do you want to proceed?

Reply:
@niradler looks good. We need to do the same for fixes, but this is good for me.